src/Parser/DemonHunter/Havoc/Modules/Abilities.js

```diff
@@ -29,7 +29,7 @@ class Abilities extends CoreAbilities {
           base: 1500,
         },
         castEfficiency: {
-          suggestion: true,
```

Comment:
isn't fury of the illidari just gone? so this could be removed from abilities entirely?

Reply:
makes totally sense. Removing it right now :D